src/Orc.SupportPackage.Example/SupportPackage/Providers/CustomSupportPackageProvider.cs

```diff
@@ -18,8 +18,12 @@ public class CustomSupportPackageProvider : SupportPackageProviderBase
             Argument.IsNotNull(() => supportPackageContext);
 
             var file = supportPackageContext.GetFile("testfile.txt");
-            
+          
             File.WriteAllText(file, "custom suppport package contents");
```

Comment:
Please use IFileService (Orc.FileSystem) and use _fileService.WriteallText instead